app/views/RefusedAmountView.scala.html

```diff
@@ -24,31 +27,32 @@
         govukInput: GovukInput,
         govukButton: GovukButton,
         continueOrExit: components.continueOrExit,
-        h1: components.h1
+        h1: components.h1,
+        itemDetailSection: itemDetailSection
 )
 
-@(form: Form[_], action: Call, unitOfMeasure: UnitOfMeasure)(implicit request: DataRequest[_], messages: Messages)
+@(form: Form[_], action: Call, unitOfMeasure: UnitOfMeasure, item: MovementItem, cnCodeInfo: CnCodeInformation, idx: Int)(implicit request: DataRequest[_], messages: Messages)
 
-@layout(pageTitle = title(form, messages("refusedAmount.title", messages(s"unitOfMeasure.$unitOfMeasure.long")))) {
+@layout(pageTitle = title(form, messages("refusedAmount.title", messages(s"unitOfMeasure.$unitOfMeasure.long"), idx))) {
 
     @formHelper(action, 'autoComplete -> "off") {
 
         @if(form.errors.nonEmpty) {
             @govukErrorSummary(ErrorSummaryViewModel(form))
         }
 
-        @h1(messages("refusedAmount.heading", messages(s"unitOfMeasure.$unitOfMeasure.long")), Some(messages("arc.subHeading", request.arc)), "govuk-heading-l")
+        @h1(messages("refusedAmount.heading", messages(s"unitOfMeasure.$unitOfMeasure.long"), idx), Some(messages("arc.subHeading", request.arc)), "govuk-heading-l")
 
         @govukInput(
             InputViewModel(
                 field = form("value"),
-                label = LabelViewModel(Text(messages("refusedAmount.heading", messages(s"unitOfMeasure.$unitOfMeasure.long")))).hidden
+                label = LabelViewModel(Text(messages("refusedAmount.heading", messages(s"unitOfMeasure.$unitOfMeasure.long"), idx))).hidden
             )
             .withWidth(Fixed10)
             .withSuffix(PrefixOrSuffix(
                 content = Text(messages(s"unitOfMeasure.$unitOfMeasure.short"))
             ))
-            .withHint(HintViewModel(Text(messages("refusedAmount.hint"))))
+            .withHint(HintViewModel(Text(messages("refusedAmount.hint")))).withHint(HintViewModel(HtmlContent(itemDetailSection("refusingAnyAmountOfItem.item", item, cnCodeInfo))))
```

Comment:
This has lost the `refusedAmount.hint` - the second part overwrites the first.

You'll need to create a block of HTML to pass into the HintViewModel. See example here:
https://github.com/hmrc/emcs-tfe-report-a-receipt-frontend/blob/main/app/views/ItemMoreInformationView.scala.html#L76-L81